_posts/2013-07-17-using-caption-services-with-html5-video.md

```diff
@@ -62,11 +74,15 @@ In addition to the VTT file format, you can also use TTML (Time Text Markup Lang
         </body>
     </tt>
 
+You may also see captions in formats like `.srt` or others that may work best for your needs.
 
 ## Transcribing
-Now that you know how to implement it, you need to transcribe all of your videos. You can watch, and rewatch, and rewatch them to capture all of the talking perfectly timed (I watched that Alice in Wonderland trailer 30 or 40 times and it is only a minute long). However, there are services that can handle the transcription for you. These will typically start at $1 per minute for transcription with a fee of $0.25 per minute for the needed timestamping.
+Now that you know how to implement it, you need to transcribe all of your videos. You can watch, and rewatch, and rewatch them to capture all of the talking perfectly timed (I watched that Alice in Wonderland trailer 30 or 40 times and it is only a minute long). One way to simplify this process is to use YouTube's auto-generated captions. Simply upload your video to YouTube (you can set it to private) and choose the auto-caption feature. Once these have been created, it should only take one or two reviews of the video to make sure the captions match and correct things like spelling and proper nouns. YouTube then lets you download the caption file in 
```

Comment:
Looks like the last sentence of this paragraph wasn't finished.

"YouTube then lets you download the caption file in"  

in what?